src/main/scala/firrtl/backends/verilog/VerilogEmitter.scala

```diff
@@ -854,10 +854,7 @@ class VerilogEmitter extends SeqTransform with Emitter {
             case MemoryLoadFileType.Binary => "$readmemb"
             case MemoryLoadFileType.Hex    => "$readmemh"
           }
-          val inlineLoad = s"""
-                              |initial begin
-                              |  $readmem("$filename", ${s.name});
-                              |end""".stripMargin
+          val inlineLoad = s"""$readmem("$filename", ${s.name});""".stripMargin
```

Comment:
```suggestion
          val inlineLoad = s"""$readmem("$filename", ${s.name});"""
```
I was literally suggesting this as you changed it 😉 , still, the `stripMargin` is unnecessary when it's one line.

Reply:
You could also just inline this whole thing if you wanted:
```scala
memoryInitials += Seq(s"""$readmem("$filename", ${s.name});""")
```

Reply:
Done, way better and cleaner.
Thanks and sorry about the noise :)